subxt/src/tx/signer.rs

```diff
@@ -25,6 +25,7 @@ pub trait Signer<T: Config> {
 }
 
 #[cfg(feature = "substrate-compat")]
+#[cfg_attr(docsrs, doc(cfg(feature = "substrate-compat")))]
```

Comment:
This line that we have to add below each feature is sortof ugly! I'm not really familiar with it; what does it do? I also wonder whether we could avoid this line duplication somehow.

Reply:
we could write a macro for it as we have in jsonrpsee

```rust
macro_rules! cfg_feature {
	($feature:literal, $($item:item)*) => {
		$(
			#[cfg(feature = $feature)]
			#[cfg_attr(docsrs, doc(cfg(feature = $feature)))]
			$item
		)*
	}
}

```